_FIPS201/requirements.md

```diff
@@ -63,9 +63,9 @@ The minimum requirement for PIV Credential eligibility determination is a comple
 
 Before an individual is determined eligible to be issued a PIV Card when no corresponding prior investigation exists, the appropriate required investigation **SHALL** be initiated with the authorized federal investigative service provider and the FBI NCHC portion of the background investigation **SHALL** be completed and favorably adjudicated.
 
-Once the investigation is completed, the authorized adjudicative entity **SHALL** adjudicate the investigation and report the final eligibility determination to the Central Verification System (or successor). This determination **SHALL** be recorded in the PIV enrollment record to reflect PIV eligibility for the PIV cardholder and, if applicable, their enrollment in the Continuous Vetting Program.
+Once the investigation is completed, the authorized adjudicative entity **SHALL** adjudicate the investigation and report the final eligibility determination to the Central Verification System (or successor). This determination **SHALL** be recorded in the PIV enrollment record to reflect PIV eligibility for the PIV cardholder and, if applicable, their enrollment in the Continuous Vetting Program as defined in EO 13467 as amended.
```

Comment:
EO 13467 does not discuss continuous vetting. I think the right reference is EO 13764.

Reply:
EO 13764 amends EO 13467.  (the anagram is messing with me whenever I read them)

I know it was recommended to reference "EO 13467 as amended," but I agree wit Jim is seems much cleaner to just reference EO 13467 directly.